### ISSUE TITLE
Create maintenance kit for workstations

### ISSUE THREAD

**Issue by RebekkaML** [NHMDenmark/Herbarium-Sheets-workstation#157]
@Chelsea Graham suggested having a maintenance kit for the different digitizing workstations that includes cleaning supply etc.

We already have a lot of useful things, but it would be good to keep them all together in a clearly labeled box close to the workstations and make sure everyone knows where it is and how to use everything.

Ideas so far:
microfiber cloths (already at HERB, but not PIOF)
cleaning agent for screens, touchpads etc.
oil (?) for PIOF sledges
air blower to remove dust from PIOF stations

**Comment by RebekkaML** [comment]
One useful part of this would be something to keep the sledges at the PIOF stations moving smoothly without making them sticky over time (https://github.com/NHMDenmark/Pinned-Insects-workstation/issues/129). Sewing machine wax worked for a time, but then formed a sticky mass with all the aluminum dust from the rails for the sledge. It's clean and working again now, but in the future we might need something to smoothen it again.

**Comment by Gomismis** [comment]
I don't think it's a good idea to ad oil to the sledges, as the resistance comes from build up of residue. but maybe something to scrape of build up residue? 
Otherwise I was thinking that a toolset for cameras including:
- umbraco key (head size 3 mm) one for each work station with a camera.  
- Zeiss lens cleaning set (just one for each location) 

I also think it's good with the cans of compressed air for pinned insects, and new note books and pens for all workstations. Maybe most important some sort of box to keep it all in so it dosen't drift :)

**Comment by chelseagraham** [comment]
I agree with everything above, especially the compressed air, lens cloths and tools for the camera brackets and copystand (keys). Maybe it is a good idea to add the extra connection cables and perhaps something (Q-tips or flat sponge sheets and scissors) for some small cleaning / application of the sewing machine oil or microscope oil?

**Comment by annejodobel** [comment]
Hi @RebekkaML, @Gomismis and @chelseagraham
So far the maintenance kit list comprises:

1. Microfiber cloths (all WSs)
2. Cleaning agents for screens and touchpads etc. (All WSs)
3. Cans of compressed air (for PIOF only?)
4. Airblower (for PIOF only?)
5. Q-tips (All WSs)
6. Flat sponge sheets (All WSs)
7. Scissors (All WSs)
8. Umbraco Key (head size 3 mm) (for cameras) (All WSs, however not associated Digiapp stations at herbariums)
9. Notebook (All WSs)
10. Pen (All WSs)
11. Sewing machine or microscope oil (For PIOF)

I'll be working on identifying suppliers for the content and finding a good storage solution (box, folder, container) in the next couple of weeks, so please add any further wishes to this ticket within that timeframe, e.g. specific tools etc.

As I think we may only need either Airblower or compressed air cans, please also comment on your preferred option, and whether oil for the PIOF sledges is still necessary since the brass replacement was installed.

**Comment by chelseagraham** [comment]
Lens cloths (if not covered by 1. Microfiber cloths)

**Comment by annejodobel** [comment]
Do we want a cleaning agent for the sledges at PIOF WS? I see that brass sledges are also starting to create this sticky dust. Potentially the best solution is to clean them with a cloth, some sort of soap and warm water ocassionally?

**Comment by annejodobel** [comment]
Chelsea suggest to consult Kim Nielsen about caring for the rails - Anne will follow up.

**Comment by annejodobel** [comment]
Kim's response:

_Jeg ved faktisk ikke hvad det bedste ville være. Ved at Anders Illum rengjorde dem i alu før jeg skiftede dem ud til messing. 
Umiddelbart tænker jeg det handler mere om at få dem affedet med sprit eller sådan noget. Og hvis man derefter har brug for slæden glider lidt bedre kunne man overveje paraffin, eller alternativt stearin som man kunne gnubbe på skinnerne, eller under slæden.

Genneralt er denne skinne/slæde løsning ikke optimal når den skal køres frem og tilbage så mange gange, men det var lige løsningen her og nu. 
Men burde nok finde nogle skuffeskinner med kuglelejer af en art._

Simply translated:
He thinks they could be 'de-oiled' using an alkoholbased cleaner (i.e. husholdningssprit) and then treated with paraffin or candle wax, to make it slide more easily again. He also points out that this is not an optimal solution, and says that it would be better with something like drawer rails with ball bearings to avoid the sticky dust issue.

Obviously the sledge system will not be changed anytime soon, if at all, but maybe we could discuss the cleaning of the rails at the next digitiser catch-up?

**Comment by RebekkaML** [comment]
@annejodobel could you update this ticket? Have the maintenance kits arrived in Aarhus yet?

**Comment by annejodobel** [comment]
All the maintenance kits are finally in place. I will make a new issue quarterly to check if any of the parts in the kits needs replacing.

Closing with this final comment.
